src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+@import url(./stars.scss);
+@use "./variables" as v;
+
+$main-color: rgb(6, 11, 53);
+$secoundary-color: rgb(97, 96, 112);
+$distance-to-border: 16px;
+
+.card {
```

Comment:
Please check height of your card, it should be less. Try to reset default margins of elements (p, h1, h2, h3, ...)
![image](https://user-images.githubusercontent.com/54936044/227362367-7b97c2dd-9803-4af1-943a-7c8ac42a2d83.png)